pipeline_config.json

```diff
@@ -36,6 +36,12 @@
       "TestContactUUIDs": [
         "61e74e0b-1b4a-41fe-9e78-712f2a048f4e"
       ]
+    },
+    {
+      "SourceType": "ShaqadoonCSV",
+      "ActivationFlowURLs": [
+        "gs://avf-project-datasets/2019/OCHA/hormud-recovery-1.csv"
```

Comment:
Please could you chat with @IsaackMwenda and agree a naming scheme for files like this. We've now seen a couple of different options on IMAQAL, and I think this is also an additional options.

In general it's going to be hard to cover all of the cases, but I think we've probably seen enough examples to have a first attempt at a naming scheme.

Reply:
Will address in the next PR

Reply:
addressed here: https://github.com/AfricasVoices/Project-OCHA/pull/47